README.md

```diff
@@ -8,31 +12,52 @@ punctuation substitutions, etc.), and it is safe for all utf-8
 (unicode) input.
 
 HTML output is currently supported, along with Smartypants
-extensions. An experimental LaTeX output engine is also included.
+extensions.
 
 It started as a translation from C of [Sundown][3].
 
 
 Installation
 ------------
 
-Blackfriday is compatible with Go 1. If you are using an older
-release of Go, consider using v1.1 of blackfriday, which was based
-on the last stable release of Go prior to Go 1. You can find it as a
-tagged commit on github.
-
-With Go 1 and git installed:
+Blackfriday is compatible with any modern Go release. With Go 1.7 and git
+installed:
```

Comment:
This is a little confusing. First it says "any modern Go release", then "Go 1.7". Can we be more clear?